code-review-tester-service/Program.cs

```diff
@@ -1,6 +1,42 @@
+using Microsoft.AspNetCore.Mvc;
+
 var builder = WebApplication.CreateBuilder(args);
 var app = builder.Build();
 
 app.MapGet("/", () => "I'm ALIVE!");
 
+app.MapGet("/tests/", (string? level) => 
+{
+    // If no level is provided, return all available levels
+    if (string.IsNullOrEmpty(level))
+    {
+        return Results.Ok(new[] { "Easy", "Medium", "Hard", "Extra Hard" });
```

Comment:
Can this return the exact contents of the Enum to avoid future changes in one breaking the other?